content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -171,20 +170,6 @@ for `kube-apiserver`, `kube-controller-manager`, `kube-scheduler` and `etcd` to
 If the flag is not set, those components run as root. You can change the value of this feature gate before
 you upgrade to a newer version of Kubernetes.
 
-`UnversionedKubeletConfigMap`
-: This flag controls the name of the {{< glossary_tooltip text="ConfigMap" term_id="configmap" >}} where kubeadm stores
-kubelet configuration data. With this flag not specified or set to `true`, the ConfigMap is named `kubelet-config`.
-If you set this flag to `false`, the name of the ConfigMap includes the major and minor version for Kubernetes
-(for example: `kubelet-config-{{< skew currentVersion >}}`). Kubeadm ensures that RBAC rules for reading and writing
-that ConfigMap are appropriate for the value you set. When kubeadm writes this ConfigMap (during `kubeadm init`
-or `kubeadm upgrade apply`), kubeadm respects the value of `UnversionedKubeletConfigMap`. When reading that ConfigMap
-(during `kubeadm join`, `kubeadm reset`, `kubeadm upgrade ...`), kubeadm attempts to use unversioned ConfigMap name first;
-if that does not succeed, kubeadm falls back to using the legacy (versioned) name for that ConfigMap.
```

Comment:
For core components, we document graduated feature gates as well as active ones.

I'd like to see that convention copied here; readers might infer that this list works the same as the main feature gate list.
One possible mechanism: use [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) to show graduated feature gates, and omit them from the normal page flow.

Reply:
updated, did not include `<details>` but kept the FG as GA in the list.